data/messages.json

```diff
@@ -19,6 +19,7 @@
   "ANNOUNCE_MINIGAME": "{CCD782}Sign up for the {838F31}%s{CCD782}! Type {838F31}%s{CCD782} to join.",
   "ANNOUNCE_NEWS_MINIGAME_JOINED": "~r~~h~%s~w~ has signed up for ~y~%s~w~ (~p~%s~w~)",
   "ANNOUNCE_REPORT": "{FFFF00}* Report by %s (Id:%d): Player: %s (Id:%d) - Reason: %s",
+  "ANNOUNCEMENT_TOGGLED":  "The {FFA500}%s{A9C4E4} announcement has been %s for you.",
```

Comment:
Should this start with @success?

Reply:
It should! I was not aware of Success. Thanks.